dbcon/execplan/treenode.h

```diff
@@ -32,6 +32,8 @@
 #include <stdlib.h>
 #include <unistd.h>
 
+#include <quadmath.h>
```

Comment:
We surely will remove quadmath from the code b/c it brings additional dependency so we shouldn't use it.

Reply:
I have removed the quadmath dependency from this file.